src/bootstrap/bootstrap.ts

```diff
@@ -4,103 +4,161 @@ import path from 'path';
 import fs from 'fs';
 import Logger, { LogLevel, StreamHandler } from '@matrixai/logger';
 
+import { DB } from '@matrixai/db';
+import { Schema } from '@/schema';
 import * as bootstrapErrors from './errors';
-import PolykeyAgent from '../PolykeyAgent';
-
-import * as errors from '../errors';
 import * as utils from '../utils';
-import * as agentUtils from '../agent/utils';
+import { KeyManager, utils as keyUtils } from '../keys';
+import { VaultManager } from '../vaults';
+import { Status } from '../status';
+import { ACL } from '../acl';
+import { GestaltGraph } from '../gestalts';
+import { Sigchain } from '../sigchain';
+import { NodeManager } from '../nodes';
+import { ForwardProxy, ReverseProxy } from '../network';
 
 async function bootstrapPolykeyState(
   nodePath: string,
   password: string,
-): Promise<void> {
-  const logger = new Logger('AgentServerTest', LogLevel.WARN, [
+  recoveryCode?: string,
+  fresh: boolean = false,
+  fastKeyGen: boolean = false,
+): Promise<string | undefined> {
+  const logger = new Logger('Bootstrapping', LogLevel.WARN, [
     new StreamHandler(),
   ]);
 
   // Checks
-  // Checking for running agent.
```

Comment:
I'm going to simplify this, as long as the directory is occupied, if it has even just 1 file/directory in it, that means `force` is needed.

However. Before you even do this, you should be checking for the agent lock. If it is locked, you must fail. This check can be done before any other checks are done.

Reply:
I think this has been addressed.